src/map/map.cpp

```diff
@@ -973,7 +973,7 @@ void map_helpscreen(int32 flag)
 
 void map_versionscreen(int32 flag)
 {
-    ShowInfo(CL_WHITE "Topaz version %d%02d_%d (%s)" CL_RESET "\n", XI_MAJOR_VERSION, XI_MINOR_VERSION, XI_REVISION, XI_RELEASE_FLAG ? "stable" : "unstable");
+    ShowInfo(CL_WHITE "Sever version %d%02d_%d (%s)" CL_RESET "\n", XI_MAJOR_VERSION, XI_MINOR_VERSION, XI_REVISION, XI_RELEASE_FLAG ? "stable" : "unstable");
```

Comment:
Apparently my one comment was lost somehow, minor, but just need an 'r' here.  Not even sure if we display this anywhere.

Reply:
woops